internal/uploader/uploader.go

```diff
@@ -0,0 +1,28 @@
+package uploader
+
+import (
+	"context"
+
+	"github.com/go-openapi/strfmt"
+	eventsapi "github.com/openshift/assisted-service/internal/events/api"
+	"github.com/openshift/assisted-service/pkg/k8sclient"
+	"github.com/sirupsen/logrus"
+	"gorm.io/gorm"
+)
+
+type UploadHandler interface {
+	Upload(ctx context.Context, clusterID *strfmt.UUID, eventsHandler eventsapi.Handler) error
```

Comment:
Are we ever going to use this interface to upload anything other than events?
Maybe it's worth making this more specific like `UploadEvents` if we're eventually going to have `UploadLogs`, for example.

Reply:
Gotcha, making it more specific is better. Modified it to `UploadEvents`!